README.md

```diff
@@ -110,7 +110,7 @@ Please cite the following publications if GauXC was used in your publication:
 
 % CUDA and distributed memory implementation
 @article{williams20on,
-  author={David B. Williams--Young and Wibe A. de Jong and Hubertus J.J. van Dam and
```

Comment:
I always style my name as an m-dash in LaTeX. This was intentional. Please revert this and the other corresponding change.

Reply:
Then the other instances need to be rectified.

Reply:
Done

Reply:
I do note that double-barreled names are typically expressed with a hyphen
https://en.wikipedia.org/wiki/Double-barrelled_name

since dashes are usually used to denote authorships, e.g. Hohenberg--Kohn and Kohn--Sham. Merriam-Webster states

https://www.merriam-webster.com/grammar/em-dash-en-dash-how-to-use

> The en dash replaces the word to between capitalized names, and is used to indicate linkages such as boundaries, treaties, and oppositions.